src/components/tabs/Tabs.vue

```diff
@@ -80,6 +112,67 @@ export default {
                 }
             ]
         }
+    },
+    methods: {
+        manageTablistKeydown(event) {
+            // https://developer.mozilla.org/fr/docs/Web/API/KeyboardEvent/key/Key_Values#Navigation_keys
+            const { key } = event
+            switch (key) {
+                case this.vertical ? 'ArrowUp' : 'ArrowLeft':
```

Comment:
I would argue that regardless of visual orientation both up/down AND left/right should be used for tab switching.

Reply:
First of all, thank you very much for your review!! I really appreciate it 😃 
According to https://www.w3.org/TR/wai-aria-practices/#h-note-25, we should keep default behavior of arrow buttons depending on tabs orientation.

Reply:
No prob. Yeah, my reasoning was more to avoid the context switch — but yeah if that's the recomendation that seems like the way to go. There are bigger fish to fry WRT a11y in here! 😄